web/src/lib.rs

```diff
@@ -81,6 +81,7 @@ struct RuffleInstance {
     has_focus: bool,
     trace_observer: Arc<RefCell<JsValue>>,
     log_subscriber: Arc<Layered<WASMLayer, Registry>>,
+    actually_used_renderer: String,
```

Comment:
Instead of storing `actually_used_renderer` in a field, I suggest adding a dedicated method for this to the `RenderBackend` trait, similar to `debug_info()`:
https://github.com/ruffle-rs/ruffle/blob/fe56f8d2120eb9d10aa72d9ff2898c46afe96b0f/render/src/backend.rs#L69

It could be something like:
```rs
fn name(&self): &'static str;
````

Reply:
I like your suggestion, and I considered that multiple times, but I chickened out because I did not want to mess too much with the other crates of the codebase and I was in doubt about whether the name values for renderer-backends used in the `web` crate https://github.com/ruffle-rs/ruffle/blob/fe56f8d2120eb9d10aa72d9ff2898c46afe96b0f/web/src/lib.rs#L1437 would also be good to use in the rest of the codebase (like, should they be kept isolated or not). But, given that you suggest it, I would like to implement it, especially if @n0samu also likes the idea.

I also believe it would also simplify the code and make it cleaner.

Should I give it a go?

Reply:
I wouldn't suggest it if I don't mean it :)
If you ask me, go for it!